source/integrations/setup-nginx-proxy-with-minio.rst

```diff
@@ -74,10 +82,25 @@ There are two models for proxying requests to the MinIO Server API and the MinIO
                proxy_set_header Connection "";
                chunked_transfer_encoding off;
 
-               proxy_pass https://minio:9000/; # This uses the upstream directive definition to load balance
+               proxy_pass https://minio/; # This uses the upstream directive definition to load balance
             }
+         }
+
+         server {
+            listen       81;
+            listen  [::]:81;
+            server_name  minio.example.net;
```

Comment:
This means that the proxy is listening on port :81 for Console calls right?

Wondering if we should reverse this:

- Since Console is web property, it should listen on `80` and then pass to `9090`
- For MinIO S3, the port I assume doesn't matter as much - can we not listen on :9000 here?